crypto/types/cache.go

```diff
@@ -0,0 +1,82 @@
+package types
+
+import (
+	lru "github.com/hashicorp/golang-lru/v2"
+)
+
+const (
+	TxHashLen        = 32
+	AddressStringLen = 2 + 20*2
+)
```

Comment:
I don't see where these two are being used. Also the length of the address was configurable by chains iirc